tasks.sh

```
@@ -80,62 +93,65 @@ function reloadCMakeProject() {
         fi
 
-        set -x
         # ref: https://stackoverflow.com/questions/11865085/out-of-a-git-console-how-do-i-execute-a-batch-file-and-then-return-to-git-conso
-        cmd "/C cmake.exe --build . $TARGET_RULE $WIN_CMAKE_BUILD_DEFINE -- /m"
-        set +x
+        cmd "/C cmake.exe --build . $TARGET_RULE $WIN_CMAKE_BUILD_DEFINE --parallel $(nproc)"
+
     else
-        set -x
+
         cmake --build . $TARGET_RULE $(test -n "$CMAKE_BUILD_TYPE" && echo --config $CMAKE_BUILD_TYPE) -- -j $(nproc)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)